support-workers/src/main/scala/com/gu/zuora/productHandlers/ZuoraDigitalSubscriptionGiftRedemptionHandler.scala

```diff
@@ -0,0 +1,86 @@
+package com.gu.zuora.productHandlers
```

Comment:
this is moved from CreateZuoraSubscription main file and minor tidying up but otherwise unchanged.